src/assets/index.js

```diff
@@ -104,6 +108,11 @@ form.addEventListener("submit", (e) => {
 
 const analyze_click = () => {
     window.history.pushState('Analyze Page', 'Crop AI', '?q=analyze');
+    var tags = ["#symptoms", "#treatment", "#products"];
+    for(var i=0 ; i<tags.length;i++){
+      const del_list = document.querySelector(tags[i]);
+      del_list.innerHTML = '';
+    }
```

Comment:
move this code to update_result function, add comments describing what this code do.